taxi/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
The setUpTestData method is a good place to set up data that will be shared among test methods. However, it's best to create actual model instances here instead of just setting up the attributes. This way, you won't have to repeat the creation of the instances in every test method.